test/e2e/data/infrastructure-aws/cluster-template.yaml

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
if we already rely on having the `aws-provider` repo existing to generate the images, can we use the template already there?

Reply:
It is possible to use files from other repository (if they exist locally) but IMO test should be as isolated as possible, so I prefer to have separate templates